commons/src/interfaces/submission/default-options.interface.ts

```diff
@@ -7,6 +7,7 @@ export interface DefaultOptions {
   tags: TagData;
   description: DescriptionData;
   rating?: SubmissionRating | string;
+  spoilerText?: string;
```

Comment:
I think this idea is fine. Though I see a lot of undefined checks for the existence of the overwrite field.
I think in this case, it might be better to create this as a object type similar to tags or description (i.e. TagData, DescriptionData) where we have an object that contains the override flag. This will avoid having to create optional and specifically named prop checks and keep things standardized across all websites.

Reply:
Fair enough I guess.
Will test this after the next version is out and merge if no issues.